teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +122,283 @@ requirements, and provenance attestations can be made to meet software supply ch
 [TEP-0091: Verified Remote Resources][tep-0091] will flesh out the details of signing, while this TEP will focus on
 surfacing the verification information and building a corpus of verified resources that users can trust.
 
+## Proposal
+
+### Ownership and Maintenance
+
+As previously discussed in [TEP-0003][tep-0003-upstream], we propose creating 
+two support tiers, `Community` and `Official`, through separate Tekton Catalogs.
+
+A Catalog is a git repository that complies with the organization contract 
+defined in [TEP-0003: Tekton Catalog Organization][tep-0003-org]. 
+
+The Community Catalog makes it easy for contributors to share resources, 
+while the Official Catalog provides a corpus of high quality resources 
+that users can rely on.
+
+Both the Community and Official Catalogs will be:
+* Owned by the [Tekton Catalog Maintainers][catalog-owners].
+* Published in the [Tekton Hub][hub].
+* Share infrastructure, such as testing and scanning tooling. 
+
+A given resource starts in the Community Catalog. Then it may get 
+promoted to the Official Catalog if:
+* It meets the [requirements of the Official Catalog](#requirements-1).
+* The [Tekton Catalog Maintainers][catalog-owners] approve its promotion. 
+
+If a resource is promoted to the Official Catalog, it can be 
+[deprecated][deprecation] in the Community Catalog.
+We will provide tooling, through [Caitlin][caitlin], to make it easier 
+to migrate from the Community Catalog to the Official Catalog.
+
+#### Tekton Community Catalog
+
+To maintain the low barrier to making contributions, users can easily 
+submit resources to the Community Catalog.
+
+The current Tekton Catalog at `tektoncd/catalog` will be reused as the 
+Community Catalog.
+
+##### Requirements
+
+1. The resource must comply with the contract defined in
+   [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+2. The resource must define an OWNER file in 
+   `/{resource-type}/{resource-name}/OWNERS`, that specifies 
+   at least one maintainer for that specific resource.
+
+Note that resources in the Community Catalog can meet some
+[requirements](#requirements-1) of resources in the Official Catalog.
+
+#### Tekton Official Catalog
+
+To provide a corpus of high quality resources that users can rely on, 
+we propose creating an Official Catalog with high maintenance, testing 
+and security standards. 
+
+The Official Catalog will be in a new repository named 
+`tektoncd/catalog-official`. 
```

Comment:
NIT: I'd love for the official catalog to be called `catalog` but I don't have a good suggestion about how a migration story would work for that, so I guess that's what it's going to be.

Reply:
that's going to be tough, if your concern is about users confusing which catalog is official...we can consider renaming `catalog` to `catalog-community` but that's an option we can explore later